test/groovy/MulticloudDeployTest.groovy

```diff
@@ -276,4 +289,81 @@ class MulticloudDeployTest extends BasePiperTest {
         assert cloudFoundryDeployRule.hasParameter('deployTool', 'cf_native')
     }
 
+    @Test
+    void multicloudParallelOnK8sTest() {
+        binding.variables.env.POD_NAME = "name"
+
+        stepRule.step.multicloudDeploy([
+            script                      : nullScript,
+            enableZeroDowntimeDeployment: true,
+            parallelExecution           : true,
+            source                      : 'file.mtar'
+        ])
+
+        assertTrue(executedOnKubernetes)
+        assertFalse(executedOnNode)
+
+        assert cloudFoundryDeployRule.hasParameter('script', nullScript)
```

Comment:
Minor: I think the following assertions are not required as there are not relevant for the aspect this test is testing.